### PR TITLE
[Semi-Modular] Courier Bags, Polychromic Bags

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
Adds courier bags, as well as polychromic versions of courier bags, satchels, backpacks, and duffel bags.
Adds the polychromic versions to character appearance setup.
Adds job type courier bags to their vendors... only some jobs.

## Why It's Good For The Game
more appearance modifications and polychromic allowing for greater individuality.

## Changelog
:cl:
add: Added courier bags
add: Added polychromic bags: backpacks, courier bags, satchels, duffel bags
add: Added polychromic bags to character appearance setup
add: Added courier bag job forms to certain vendors
/:cl:

### CODE REVIEW

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -128,7 +128,12 @@ GLOBAL_LIST_INIT(security_depts_prefs, list(SEC_DEPT_RANDOM, SEC_DEPT_NONE, SEC_
 #define DBACKPACK "Department Backpack"
 #define DSATCHEL "Department Satchel"
 #define DDUFFELBAG "Department Duffel Bag"
-GLOBAL_LIST_INIT(backbaglist, list(DBACKPACK, DSATCHEL, DDUFFELBAG, GBACKPACK, GSATCHEL, GDUFFELBAG, LSATCHEL))
+// SKYRAT EDIT: Courier Bags and Polychromics
```
Comment (Akrilla) [inline code/_globalvars/lists/flavor_misc.dm:131]: can you do skyrat start/end for this block

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 2]
```diff
@@ -28,6 +28,8 @@
 /obj/structure/closet/wardrobe/black/PopulateContents()
 	for(var/i in 1 to 3)
 		new /obj/item/clothing/under/color/black(src)
+	for(var/i in 1 to 3)
+		new /obj/item/storage/backpack/courier/black(src)
```
Comment (Akrilla) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:32]: skyrat change

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 3]
```diff
@@ -113,6 +115,8 @@
 /obj/structure/closet/wardrobe/grey/PopulateContents()
 	for(var/i in 1 to 3)
 		new /obj/item/clothing/under/color/grey(src)
+	for(var/i in 1 to 3)
```
Comment (Akrilla) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:118]: skyrat change

File: code/modules/jobs/job_types/_job.dm [hunk 4]
```diff
@@ -207,6 +207,15 @@
 			back = satchel //Department satchel
 		if(DDUFFELBAG)
 			back = duffelbag //Department duffel bag
+		// SKYRAT EDIT: Courier Bags and Polychromics
```
Comment (Akrilla) [inline code/modules/jobs/job_types/_job.dm:210]: skyrat start/end for this block

File: code/modules/vending/wardrobes.dm [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
 	desc = "A vending machine for security and security-related clothing!"
 	icon_state = "secdrobe"
 	product_ads = "Beat perps in style!;It's red so you can't see the blood!;You have the right to be fashionable!;Now you can be the fashion police you always wanted to be!"
 	vend_reply = "Thank you for using the SecDrobe!"
 	products = list(/obj/item/clothing/suit/hooded/wintercoat/security = 2,
-					/obj/item/storage/backpack/security = 2,
-					/obj/item/storage/backpack/satchel/sec = 2,
-					/obj/item/storage/backpack/duffelbag/sec = 3,
+					/obj/item/storage/backpack/security = 5,
+					/obj/item/storage/backpack/satchel/sec = 5,
```
Comment (Akrilla) [inline code/modules/vending/wardrobes.dm:12]: skyrat start/end for this block

**Comment by Akrilla** [comment]
etc

File: code/_globalvars/lists/flavor_misc.dm [hunk 6]
```diff
@@ -128,7 +128,12 @@ GLOBAL_LIST_INIT(security_depts_prefs, list(SEC_DEPT_RANDOM, SEC_DEPT_NONE, SEC_
...
+#define PLYSBP "Polychromic Backpack"
+#define PLYSS "Polychromic Satchel"
+#define PLYSCB "Polychromic Courier Bag"
+#define PLYSDB "Polychromic Duffel Bag"
+GLOBAL_LIST_INIT(backbaglist, list(DBACKPACK, DSATCHEL, DDUFFELBAG, GBACKPACK, GSATCHEL, GDUFFELBAG, LSATCHEL, PLYSBP, PLYSS, PLYSCB, PLYSDB))
```
Comment (jjpark-kb) [inline code/_globalvars/lists/flavor_misc.dm:136]:
```suggestion
// SKYRAT EDIT: Courier Bags and Polychromics
#define PLYSBP "Polychromic Backpack"
#define PLYSS "Polychromic Satchel"
#define PLYSCB "Polychromic Courier Bag"
#define PLYSDB "Polychromic Duffel Bag"
GLOBAL_LIST_INIT(backbaglist, list(DBACKPACK, DSATCHEL, DDUFFELBAG, GBACKPACK, GSATCHEL, GDUFFELBAG, LSATCHEL, PLYSBP, PLYSS, PLYSCB, PLYSDB))
// SKYRAT EDIT CLOSE
```

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm (hunk 2 above)
Comment (jjpark-kb) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:32]:
```suggestion
// SKYRAT EDIT: Courier Bags
	for(var/i in 1 to 3)
		new /obj/item/storage/backpack/courier/black(src)
// SKYRAT EDIT CLOSE
```

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 7]
```diff
@@ -113,6 +115,8 @@
...
+		new /obj/item/storage/backpack/courier(src)
```
Comment (jjpark-kb) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:119]:
```suggestion
// SKYRAT EDIT: Courier Bags
	for(var/i in 1 to 3)
		new /obj/item/storage/backpack/courier(src)
// SKYRAT EDIT CLOSE
```

File: code/modules/jobs/job_types/_job.dm [hunk 8]
```diff
@@ -207,6 +207,15 @@
...
+		if(PLYSBP)
+			back = /obj/item/storage/backpack/polychromic //Polychromic Backpack
+		if(PLYSS)
+			back = /obj/item/storage/backpack/satchel/polychromic //Polychromic Satchel
+		if(PLYSCB)
+			back = /obj/item/storage/backpack/courier/polychromic //Polychromic Courier bag
+		if(PLYSDB)
+			back = /obj/item/storage/backpack/duffelbag/polychromic //Polychromic Duffel bag
```
Comment (jjpark-kb) [inline code/modules/jobs/job_types/_job.dm:218]:
```suggestion
		// SKYRAT EDIT: Courier Bags and Polychromics
		if(PLYSBP)
			back = /obj/item/storage/backpack/polychromic //Polychromic Backpack
		if(PLYSS)
			back = /obj/item/storage/backpack/satchel/polychromic //Polychromic Satchel
		if(PLYSCB)
			back = /obj/item/storage/backpack/courier/polychromic //Polychromic Courier bag
		if(PLYSDB)
			back = /obj/item/storage/backpack/duffelbag/polychromic //Polychromic Duffel bag
		// SKYRAT EDIT CLOSE
```

File: code/modules/vending/clothesmate.dm [hunk 9]
```diff
@@ -150,7 +150,12 @@
 					/obj/item/clothing/under/costume/qipao/red = 3,
 					/obj/item/clothing/under/costume/cheongsam = 3,
 					/obj/item/clothing/under/costume/cheongsam/white = 3,
-					/obj/item/clothing/under/costume/cheongsam/red = 3)
+					/obj/item/clothing/under/costume/cheongsam/red = 3,
+					// SKYRAT EDIT: Courier Bags and Polychromics
+					/obj/item/storage/backpack/courier/polychromic = 3,
+					/obj/item/storage/backpack/polychromic = 3,
+					/obj/item/storage/backpack/satchel/polychromic = 3,
+					/obj/item/storage/backpack/duffelbag/polychromic = 3)
```
Comment (jjpark-kb) [inline code/modules/vending/clothesmate.dm:158]:
```suggestion
					// SKYRAT EDIT: Courier Bags and Polychromics
					/obj/item/storage/backpack/courier/polychromic = 3,
					/obj/item/storage/backpack/polychromic = 3,
					/obj/item/storage/backpack/satchel/polychromic = 3,
					/obj/item/storage/backpack/duffelbag/polychromic = 3)
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 10]
```diff
@@ -1,16 +1,17 @@
...
+					/obj/item/storage/backpack/duffelbag/sec = 5,
+					/obj/item/storage/backpack/courier/sec = 5,
 					/obj/item/clothing/under/rank/security/officer = 5,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:15]:
```suggestion
					/obj/item/storage/backpack/courier/sec = 5,
					// SKYRAT EDIT CLOSE
					/obj/item/clothing/under/rank/security/officer = 5,
```

File: code/modules/vending/wardrobes.dm [hunk 11]
```diff
@@ -1,16 +1,17 @@
...
 	product_ads = "Beat perps in style!;It's red so you can't see the blood!;You have the right to be fashionable!;Now you can be the fashion police you always wanted to be!"
 	vend_reply = "Thank you for using the SecDrobe!"
 	products = list(/obj/item/clothing/suit/hooded/wintercoat/security = 2,
-					/obj/item/storage/backpack/security = 2,
-					/obj/item/storage/backpack/satchel/sec = 2,
-					/obj/item/storage/backpack/duffelbag/sec = 3,
+					/obj/item/storage/backpack/security = 5,
+					/obj/item/storage/backpack/satchel/sec = 5,
+					/obj/item/storage/backpack/duffelbag/sec = 5,
+					/obj/item/storage/backpack/courier/sec = 5,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:14]:
```suggestion
					/obj/item/storage/backpack/courier/sec = 5,
// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 12]
```diff
@@ -37,6 +39,7 @@
 					/obj/item/storage/backpack/duffelbag/med = 5,
 					/obj/item/storage/backpack/medic = 5,
 					/obj/item/storage/backpack/satchel/med = 5,
+					/obj/item/storage/backpack/courier/med = 5,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:42]:
```suggestion
					/obj/item/storage/backpack/courier/med = 5,
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 13]
```diff
@@ -72,6 +76,7 @@
 					/obj/item/storage/backpack/duffelbag/engineering = 3,
 					/obj/item/storage/backpack/industrial = 3,
 					/obj/item/storage/backpack/satchel/eng = 3,
+					/obj/item/storage/backpack/courier/engi = 3,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:79]:
```suggestion
					/obj/item/storage/backpack/courier/engi = 3,
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 14]
```diff
@@ -95,6 +101,7 @@
 					/obj/item/storage/backpack/duffelbag/engineering = 3,
 					/obj/item/storage/backpack/satchel/eng = 3,
 					/obj/item/storage/backpack/industrial = 3,
+					/obj/item/storage/backpack/courier/engi = 3,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:104]:
```suggestion
					/obj/item/storage/backpack/courier/engi = 3,
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 15]
```diff
@@ -155,6 +163,7 @@
 					/obj/item/clothing/head/beret/sci = 3,
 					/obj/item/storage/backpack/science = 3,
 					/obj/item/storage/backpack/satchel/tox = 3,
+					/obj/item/storage/backpack/courier/sci = 3,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:166]:
```suggestion
					/obj/item/storage/backpack/courier/sci = 3,
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 16]
```diff
@@ -175,6 +185,7 @@
 	vend_reply = "Thank you for using the Hydrobe!"
 	products = list(/obj/item/storage/backpack/botany = 3,
 					/obj/item/storage/backpack/satchel/hyd = 3,
+					/obj/item/storage/backpack/courier/hydr = 3,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:188]:
```suggestion
					/obj/item/storage/backpack/courier/hydr = 3,
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 17]
```diff
@@ -366,6 +378,7 @@
 					/obj/item/storage/backpack/chemistry = 3,
 					/obj/item/storage/backpack/satchel/chem = 3,
 					/obj/item/storage/bag/chemistry = 3,
+					/obj/item/storage/backpack/courier/chem = 3,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:381]:
```suggestion
					/obj/item/storage/backpack/courier/chem = 3,
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 18]
```diff
@@ -404,12 +418,13 @@
 					/obj/item/clothing/suit/toggle/labcoat/virologist = 3,
 					/obj/item/clothing/mask/surgical = 3,
 					/obj/item/storage/backpack/virology = 3,
-					/obj/item/storage/backpack/satchel/vir = 3)
+					/obj/item/storage/backpack/satchel/vir = 3,
+					/obj/item/storage/backpack/courier/viro = 3)
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:422]:
```suggestion
					/obj/item/storage/backpack/courier/viro = 3)
					// SKYRAT EDIT CLOSE
```

File: code/modules/vending/wardrobes.dm [hunk 19]
```diff
@@ -422,6 +438,7 @@
 					/obj/item/storage/backpack/captain = 1,
 					/obj/item/storage/backpack/satchel/cap = 1,
 					/obj/item/storage/backpack/duffelbag/captain = 1,
+					/obj/item/storage/backpack/courier/com = 1,
```
Comment (jjpark-kb) [inline code/modules/vending/wardrobes.dm:441]:
```suggestion
					/obj/item/storage/backpack/courier/com = 1,
					// SKYRAT EDIT CLOSE
```

**Pull request merged**